### ISSUE TITLE
Fluctuance on palpation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10097]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Yielding to alternate pressure by palpating fingers so as to suggest that the area being felt contains fluid

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Breast abscess

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000062
- label: Fluctuance on palpation
- parents: [Clinical modifier (HP:0012823)](https://hpo.jax.org/app/browse/term/HP:0012823)
- pmid: 
- def: Yielding to alternate pressure by palpating fingers so as to suggest that the area being felt contains fluid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10097
- synonyms:
